src/main/resources/en-GB.yml

```diff
@@ -36,6 +36,57 @@ en-GB:
         - "(016064)-####"
         - "(016064)-#####"
 
+      formats_international:
+        - "+44 20-####-####"
+        - "+44 24-7###-####"
+        - "+44 29-####-####"
+        - "+44 113-###-####"
+        - "+44 114-###-####"
+        - "+44 121-###-####"
+        - "+44 131-###-####"
+        - "+44 141-###-####"
+        - "+44 151-###-####"
+        - "+44 161-###-####"
+        - "+44 1642-##-####"
+        - "+44 1223-##-####"
+        - "+44 1257-##-####"
+        - "+44 1382-##-####"
+        - "+44 1386-##-####"
+        - "+44 1935-##-####"
+        - "+44 1865-##-####"
+        - "+44 1772-##-####"
+        - "+44 1792-##-####"
+        - "+44 1204-##-####"
+        - "+44 15396-##-###"
+        - "+44 16977-#####"
+        - "+44 16064-####"
+        - "+44 16064-#####"
+
+      formats_national:
```

Comment:
so, why do we have formats_national and formats now? Also, wouldn't it make sense to base the formats_international on the formats_national?

Reply:
@bodiam I updated the code according to your advice